src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  background-color: #50464634;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__item {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin-right: 20px;
+  text-transform: uppercase;
+}
+
+.nav__item.lastitem {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: flex;
+  position: absolute;
+  width: 37px;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
Why? It is for pseudo element, if I will do like that element will be on all width of nav

Reply:
<img width="176" alt="Screenshot 2024-08-16 at 17 09 03" src="https://github.com/user-attachments/assets/a6e86069-604a-4c38-8cc3-507d1478e345">
It should work for all links